src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4385 @@
+4. Set **Camera Collision** to **Force Off** for the **Cube** object.
+5. Rename **Cube** to **Position**.
+6. Duplicate and space these out along the starting line.
+7. Set **Visibility** to **Force Off** for all **Position** objects.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/create_start_positions.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Setup Script Hierarchy
+
+We need to setup the script folder and contexts in the **Hierarchy**.
+
+1. Create a folder called **Scripts** inside the **Race Timer Tutorial - Track** group.
+2. Create a **Client Context** inside the **Scripts** folder.
+3. Create a **Server Context** inside the **Scripts** folder.
+
+### Client Context vs Server Context
+
+Understanding the difference between these two contexts can help you understand where to place **Lua** scripts and what they should be used for.
+
+Anything in the **Client Context** exists on the clients version of the game. This means that there is no network traffic, and anything that gets modified is applied to that local player and not all players on the server.
+
+With **Lua** scripts, nothing sensitive should be stored in a **Client Context** script, and any action the client can do should always be sent to a script in a **Server Context** to validate it. **The client should never be trusted.**
+
+Take a look at the example below of what is bad vs good.
+
+```lua linenums="1" hl_lines="9"
+-- Client script
+
+local itemPrice = 1500
+local itemId = 3
+
+-- Client can purchase an item from the game shop.
+
+shop_button.clickedEvent:Connect(function()
+    Events.BroadcastToServer("purchase_item", itemPrice, itemId)
+end)
+```
+
+In the above script, see line 9 where the price and the ID of the shop item is being sent to the server. This is bad because the broadcast to the server is also sending the price of the item which could be modified by the client.
+
+```lua linenums="1" hl_lines="9"
+-- Client script
+
+local itemId = 3
+
+-- Client can purchase an item from the game shop.
+-- Notice that no price is sent to the server.
+
+shop_button.clickedEvent:Connect(function()
+    Events.BroadcastToServer("purchase_item", itemId)
+end)
+```
+
+In the above script, see line 9 where the item ID is sent to the server only. The server script should then handle validating if the player can purchase the item, and if so it will initiate the purchase and possibly broadcast back to the client if it was successful or not.
+
+!!! tip
+    There is another context called **Default Context**. Any script in this context is also viewable by the client. So if you have code you want to protect from prying eyes, consider moving it to a **Server Context**.
+
+For further information checkout [Networking in Core](../references/networking/)
+
+## Race Manager Server Script
```

Comment:
Start h2 sections with a gerund ... "Creating the Race Manager Server Script" or building/designing/writing